### PR TITLE
Enable quantization-aware training (QAT) by default

### PR DESCRIPTION
**Pull request by rotx-maxim**
Replacement for #36

### CODE REVIEW

**Review by vicloginov — APPROVED** [review]

File: quantize.py [hunk 1]
```diff
@@ -20,13 +20,21 @@
 from devices import device
 
 CONV_SCALE_BITS = 8
+CONV_DEFAULT_WEIGHT_BITS = 8
 FC_SCALE_BITS = 8
 FC_CLAMP_BITS = 8
 
-DEFAULT_SCALE = .85
+DEFAULT_SCALE = 1.0
```
Comment (aniktash) [inline quantize.py:27]: is the scale used in QAT as well?

Comment (aniktash) [reply quantize.py:27]: If QAT is disabled, the same 0.85 default is expected.

**Review by vicloginov — DISMISSED** [review]

File: quantize.py (hunk 1 above)
Comment (rotx-maxim) [reply quantize.py:27]: Good point. Checking with IST.

Comment (rotx-maxim) [reply quantize.py]: Going back to .85 for the default.

**Pull request merged**